### ISSUE TITLE
Latitude Financial breach impacts 225,000 customers

### ISSUE THREAD

**Issue by swidup** [vz-risk/VCDB#19601]
https://www.itnews.com.au/news/latitude-financial-breach-impacts-225000-customers-592152

**Comment by swidup** [comment]
https://www.abc.net.au/news/2023-03-18/latitude-financial-customers-furious-over-data-breach/102112474

**Comment by swidup** [comment]
https://eftm.com/2023/03/latitude-financial-begins-notifying-customers-of-data-breach-up-to-328000-customers-affected-232459

**Comment by swidup** [comment]
Harvey Norman, JB Hi-fi customers hit in Latitude finance hack https://www.smh.com.au/business/companies/harvey-norman-jb-hi-fi-customers-hit-in-latitude-finance-hack-20230319-p5ctgc.html

**Comment by swidup** [comment]
https://www.cnn.com/2023/03/26/australia/latitude-group-information-theft-intl-hnk/index.html 8 million?  That's quite different.

**Comment by swidup** [comment]
https://www.theguardian.com/australia-news/2023/mar/27/latitude-financial-cyber-data-breach-hack-14m-customer-records-stolen or is it 14 total?

https://www.hackread.com/latitude-financial-data-breach/

**Comment by swidup** [comment]
GE Money https://www.abc.net.au/news/2023-04-14/latitude-financial-cyber-hack-ge-money-credit-card-class-action/102217242

**Comment by etgifford** [comment]
https://www.abc.net.au/news/2023-03-28/latitude-financial-customers-frustrated-lack-of-communication/102151166
https://ia.acs.org.au/article/2023/data-breach-cost-latitude--76-million.html#:~:text=The%20breach%20of%20the%20company,from%20two%20other%20service%20providers.
https://ia.acs.org.au/article/2023/latitude-financial-hacked--328-000-records-stolen.html
https://www.asx.com.au/asxpdf/20230316/pdf/45mqrr4z1tb693.pdf  Latitude announcement of the hack
https://www.scmagazine.com/brief/data-breach-costs-latitude-financial-nearly-50m
https://cdn-api.markitdigital.com/apiman-gateway/ASX/asx-research/1.0/file/2924-02669910-3A619038?access_token=83ff96335c2d45a094df02a206a39ff4 Latitude slide presentation on the hack

**Comment by etgifford** [comment]
Latitude Financial Services, HQ: Melbourne, Victoria, AU, EMP: 1198, NAICS: 522291, (personal finance products, credit cards, personal loans, car loans, insurance) services customers in Australia and New Zealand, announced Mar 20, 2023 it had a security breach of 14MM records Mar 16, 2023: driver’s licenses, passport numbers, financial statements, Medicare numbers, name, address, phone number, date of birth, income and expense information, bank account numbers, credit card and credit card account numbers. Routine scans on the network flagged suspect activity on an administrator account. Latitude revealed that the attackers gained employee login credentials through an attack on a third party (understood to be DXC technology), and used them to steal personal information from two other service providers. Latitude Financial refused to pay a ransom request not to sell the information on the Dark Web. The incident cost the company $53MM according to its financial statements. Latitude also confirmed that 6.1 million customer records that were provided before 2013 were compromised in the hack, including some dating back to 2005. 
Side Note: GE Capital/GE Money was mentioned in https://www.abc.net.au/news/2023-04-14/latitude-financial-cyber-hack-ge-money-credit-card-class-action/102217242 and Heartbleed from 2014. I built the GE Capital CIRT and ran it until GE Capital divestiture. I worked this Heartbleed incident in 2014. No complete customer PII was exfiltrated i.e., there were parts, but not enough to put together what information goes to what customer.
